pkg/fixtures/example_project/foo/collision.go

```diff
@@ -0,0 +1,7 @@
+package foo
+
+import "github.com/vektra/mockery/v2/pkg/fixtures/example_project/bar/foo"
+
+type Collision interface {
```

Comment:
Maybe add a comment in this describing what the intended situation here is (obviously, package and import having the same name).

Reply:
I renamed the interface, so now it has a more explicit name

Reply:
Thanks @i-sevostyanov , tests seem to be failing unfortunately. I'll let you take care of that.

Reply:
Oh, sorry for that. Fixed 🤓